scss/civicrm/contact/pages/_contact-dedupefind.scss

```diff
@@ -0,0 +1,48 @@
+/* stylelint-disable max-nesting-depth, selector-max-compound-selectors, selector-no-qualifying-type, selector-max-id */
+
+#{civi-page('contact-dedupefind')} {
+  .crm-container {
+    .crm-search_filters-accordion {
+      @include box-shadow($box-shadow);
+      margin-bottom: 20px;
+    }
+
+    .form-item {
+      table.form-layout-compressed {
+        margin: 0 !important;
+      }
+
+      #dupePairs_wrapper {
+        margin-top: 20px;
+
+        .dataTables_length,
+        #dupePairs_filter {
+          padding: 15px;
+        }
+
+        .dataTables_info {
+          padding: 12px;
+        }
+
+        #dupePairs_filter {
+          input[type=search] {
+            display: inline-block;
+            width: 230px;
+          }
+        }
+
+        tr.columnheader {
+          border: 0;
+
+          th {
+            border-bottom-color: $crm-grayblue-dark;
+          }
+        }
+      }
+    }  
+  }
+
+  #{civi-dialog()} {
+    min-width: 400px;
```

Comment:
why do we have to customize the width of the dialog just for this page?

Reply:
Based on the fonts and style we are using, this modal is cutting of title, so, best and easiest solution for this giving it proper height so, it doesn't look too small and have title visibly appropriate

Screenshot without width:
![image](https://user-images.githubusercontent.com/26058635/37820915-1f4af85a-2ea8-11e8-8424-643386050949.png)